draft-ietf-aipref-vocab.md

```diff
@@ -165,56 +165,61 @@ The process of obtaining preferences has very limited scope
 for variation between implementations.
 
 
-## Respecting Preferences
+## Enforcement
 
-Specification conformance does not encompass whether preferences are actually respected
-during data processing.
-A data processor MAY choose to respect preferences that is has discovered,
-according to:
+This specification does not provide any enforcement mechanism,
+and conformance to it does not encompass whether
+preferences are actually respected during data processing.
 
-* an understanding of the nature of the processing being performed
-  and how it corresponds to the usage categories
-  where preferences have been expressed, and
+Other mechanisms
+-- technical, legal, or otherwise --
+might enforce stated preferences.
+As such they will determine the consequences for following or not following a stated preference.
+Preferences do not imply control, either in the positive or negative.
+For example, a stated preference allowing a use does not
+(on its own) imply a right to use it for that purpose;
+likewise, a stated preference disallowing a use does not
+(on its own) mean that the content cannot be used for that purpose.
 
-* the applicable legal context; see {{legal}}.
 
-Usage preferences can be overridden through express agreements
-between relevant parties.
+## Applicability and Legal Effect {#legal}
 
-There are also many situations where other priorities could override
-any usage preferences.
-Priorities that could justify ignoring preferences include --
-but are not limited to --
-free expression, safety, education, scholarship, research,
-preservation, interoperability, and accessibility.
+This document provides a set of definitions for different categories of use,
+plus a system for associating simple preferences to each
+(allow, disallow, or no preference; see {{model}}).
 
-A choice to ignore a preference could be
-explicitly permitted in law
-or be based on the judgement of particular individuals or organizations.
+The categories of use that are defined as part of the vocabulary
+are not always clearly applicable or inapplicable to a particular system or application.
+The universe of possible systems is far more complex
+than any simple vocabulary is capable of describing.
+That means that some discretion could be involved
+in deciding whether a preference applies.
 
-The following lists examples of cases
-where other priorities could override specific preferences:
+The expression of preferences might activate regulatory or legal consequences,
+which has implications for entities that consume those preferences.
+Their interpretation of the meaning of different terms
+could have legal ramifications.
+Different jurisdictions could reach subtly different conclusions
+about the applicability of each category of use
+to specific applications.
```

Comment:
This is probably most redundant, but there is still a useful point: how a category applies to a given application could be a judgment call and that might vary between jurisdictions (if there is a legal effect).